core/events/services/classic/models.py

```diff
@@ -66,6 +73,26 @@ class Submission(Base):    # type: ignore
 
     WITHDRAWN_FORMAT = 'withdrawn'
 
+    # Map classic status to Submission domain status.
+    STATUS_MAP = {
```

Comment:
This dict maps status codes in the classic DB to the (simpler) statuses in this package. Does this look right?

Reply:
`NEXT_PUBLISH` or `NEXT_PUBLISH_DAY` might be more clearer here, since "tomorrow" is not necessarily when the submission gets published.

Reply:
Looks like all the pertinent statuses are covered here 👍

Reply:
Good idea, changing